cache/memcached_client.go

```diff
@@ -28,6 +28,7 @@ import (
 
 const (
 	dnsProviderUpdateInterval = 30 * time.Second
+	maxTTL                    = 30 * 24 * time.Hour
```

Comment:
where does this come from? number of seconds in an in32? hmm, no, that's way more.